bluetooth_mesh/models/models.py

```diff
@@ -1581,6 +1581,29 @@ async def get_lightness(
             for node, result in results.items()
         }
 
+    async def set_lightness_unack(
+            self,
+            destination: int,
+            app_index: int,
+            lightness: int,
+            *,
+            retransmissions: int = 6,
+            send_interval: float = 0.075,
+    ) -> None:
+        async def request():
+            ret = self.send_app(
+                destination,
+                app_index=app_index,
+                opcode=LightLightnessOpcode.LIGHTNESS_SET_UNACKNOWLEDGED
+                params=dict(lightness=lightness)
```

Comment:
Hm, what about delay? These are application retries, so let's try to avoid the popcorn

Reply:
Also, I think the event might include transition time

Reply:
I've just added it.

Reply:
Fixed delay handling as well.